Swift/BRCrypto/BRCryptoNetwork.swift

```diff
@@ -292,3 +311,60 @@ public final class NetworkFee: Equatable {
         return CRYPTO_TRUE == cryptoNetworkFeeEqual (lhs.core, rhs.core)
     }
 }
+
+///
+/// A NetworkPeer is a Peer on a Network.  This is optionally used in Peer-to-Peer modes to
+/// specfify one or more peers to connect to for network synchronization.  Normally the P2P
+/// protocoly dynamically discovers peers and thus NetworkPeer is not commonly used.
+///
+public final class NetworkPeer: Equatable {
+    // The Core representation
+    internal let core: BRCryptoPeer
+
+    /// The network
+    public let network: Network
+
+    /// The address
+    public let address: String
+
+    /// The port
+    public let port: UInt16
+
+    /// The public key
+    public let publicKey: String?
+
+//    /// The serialization to allow reconstructing this peer
+//    public var serialization: Data {
+//        var bytesCount: Int = 0
+//        let bytes = cryptoPeerSerialize(core, &bytesCount)
+//        return Data (bytes: bytes!, count: bytesCount)
+//    }
+
+    internal init (core: BRCryptoPeer, take: Bool) {
```

Comment:
Do we need a `cryptoPeerGive()`?

Reply:
You have no `defer { cryptoPeerGive() }` in the Swift. Is that not required?

Reply:
Done.